include/zephyr/linker/llext-sections.ld

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+	/*
+	 * Special section used by LLEXT if CONFIG_LLEXT_EXPORT_BUILTINS_BY_SLID
+	 * is enabled. Declare this section to prevent it from being considered orphan.
+	 *
+	 * This section is used to temporarily save the exported symbols' names in the
+	 * Zephyr ELF for post-processing, but it is not included in the final binary.
+	 *
+	 * NOTE: This section MUST start at address 0, as the post-processing scripts
+	 * assume that the address of any data in this section (i.e., symbol names) is
+	 * strictly equivalent to the offset inside the section.
+	 */
```

Comment:
Maybe this should not be indented?

Reply:
I indented it to follow what appears to be a convention, such as in
https://github.com/zephyrproject-rtos/zephyr/blob/d77dc62d196b8f5f58622a05414002cbf40285ba/include/zephyr/linker/rel-sections.ld#L1-L35 or https://github.com/zephyrproject-rtos/zephyr/blob/d77dc62d196b8f5f58622a05414002cbf40285ba/include/zephyr/linker/kobject-text.ld#L1-L15